docs/Queries.md

```diff
@@ -90,8 +121,14 @@ A query response is also a plain object
   ref: string,
   value: any,
   type?: string,
-  flags?: array,
-  meta?: object,
+  meta?: {
+    flags?: string[],
+    variants?: {
+      [string]: {  // experiment
+        [string]: string  // context: variant
```

Comment:
Yeah, I think `[string|number]` sounds right to me. I can definitely foresee at some point adding a context that isn't a number.